src/options.ts

```diff
@@ -26,6 +27,14 @@ export interface Options {
 
   rc: RC;
 
+  // Hosting-specific (including web frameworks) options
+  normalizedHostingConfig?: HostingResolved[];
+  site?: string; // When using a single hosting site, this may be filled by firedata
+  expires?: `${number}${"h" | "d" | "m"}`;
+
+  // Emulator options:
+  port?: number;
+
```

Comment:
Comment got lost/outdated:

So I think we should avoid putting [product]-specific things into the Options object here. My thoughts before were to make sure the hosting deploy path could specify their requirements in their functions in a way that Options plus something (e.g. extending Options) could satisfy the requirement.

My fear here is that `port` shows up in Options and starts being used. If I see `options.expires`, I'm going to have no idea what that's for.

Reply:
Per offline conversation (which admittedly I'm quoting here before you had much of an opportunity to contribute) I repurposed `MockableOptions` as `HostingOptions`. I still pretty strongly believe I should leave the fields in the top-level `Options` object so that I can be protected from drift with the `Implements` assertion and so that I don't need to cast `Options as unknown as HostingOptions`. If you really strongly believe I should leave fields undocumented and violate the type system to do so, I'll grumble but comply.